_pages/en/mandate.md

```diff
@@ -16,43 +16,43 @@ The [Canadian Experience](https://github.com/sara-sabr/ITStrategy/blob/master/wi
-- Measure progress of adoption of strategies implementation in IITB teams and governance bodies as a performance indicator for the IT Strategy team
+- Diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours
+- Recommend specific course of actions to ESDC governance bodies to adopt new methods of managing IT risks
+- Measure progress of adoption of strategies in IITB teams and governance bodies as a performance indicator of the IT Strategy team
 - Challenge current thinking and approaches
 
 ## Team's Guiding Principles
 
 ### Work collaboratively
 
 Collaborate with anyone and everyone who can _add value_.
-This includes, but is not limited to, all Canadians, all levels of government, all departments, all private sector organizations, and all governments.
+This includes, but is not limited to, all Canadians, levels of government, departments, private sector organizations, and governments.
 
 ### Continuous learning
 
-Learning must be a core part of daily activities to ensure up to date, modern, and well researched direction and strategy is provided to guide our actions.
+Learning is a core part of daily activities to ensure up to date, modern, and well-researched direction.
 
 ### Lead by Example
 
-By using the approaches and technologies we champion, we will be able to better respond to, and anticipate, the challenges that will be faced.
-We will also be experimenting with modern ways of working in order to demonstrate their value and effectiveness.
+By using the approaches and technologies we champion, we are able to better respond and anticipate potential challenges.
+We also experiment with modern ways of working in order to demonstrate their value and effectiveness.
 
-## Team's understanding of the "digital transformation"
+## Team understanding of the "digital transformation"
 
-IT is now used to delivery services to Canadians. If IT fails, business fails and vice versa.
+IT is now used to deliver services to Canadians. If IT fails, business fails and vice versa.
 The use of technology is no longer "an IT problem" as it has real repercussions to citizens.
 IT can no longer be seen as a company within another company (or an outside supplier), it has shared accountability with the organization's mandate.
 
-Current IT processes and methods are risky and costly which translates into lengthy governance and compliance checks.
+Current IT processes and methods are risky and costly, which translates into lengthy governance and compliance checks.
 We must change how we do IT in order to reduce its risks, improve cyber security, and increase our ability to be flexible, so that IT can become a strategic asset for ESDC, and for citizen experience.
 Service lines must therefore also accept the risks and costs of doing IT, because IT is how ESDC successfully delivers its mandate.
 
-That is Digital Transformation.
+That is **Digital Transformation**.
```

Comment:
What is?  Service lines must therefore also accept the risks and costs of doing IT?

What is our understanding of what needs to happen?  I like the end: ongoing venture, has no end, never ends.

We should lead into it with talk about continuous improvement.

I'll make a commit with changes.